services/pypi/pypi-framework-versions.service.js

```diff
@@ -80,15 +80,15 @@ export default class PypiFrameworkVersion extends PypiBase {
     }
   }
 
-  async handle({ frameworkName, packageName }) {
+  async handle({ frameworkName, packageName }, { pypiBaseUrl }) {
```

Comment:
we'll need to also add `pypiBaseUrlParam` to the openApi spec on this service

Reply:
Thanks for your check, this is done now.